src/data/set/intervals/basic.lean

```diff
@@ -549,6 +549,30 @@ eq_singleton_iff_unique_mem.2 ⟨left_mem_Ici, λ b, h.eq_of_ge⟩
 
 lemma _root_.is_min.Iic_eq (h : is_min a) : Iic a = {a} := h.to_dual.Ici_eq
 
+lemma Ici_eq_Ici_iff : Ici a = Ici b ↔ a = b :=
```

Comment:
Can you add the version that says `function.injective (Iic : α \to set α)` too? If you prove that first, then this one can be proved with `Iic_injective.eq_iff`

Reply:
Done, thanks.